### ISSUE TITLE
Chapter2: model fitting runs only on 1875 training records(not on 60,000) of fashion_mnist from keras

### ISSUE THREAD

**Issue by SunnyMakode** [lmoroney/tfbook#12]
Recently read chapter 2 and tried the code provided. I am a bit confused about why the model is only fitting 1875 records per epoch while in the book it is 60,000 per epoch.

I also measure the length for this fashion_mnist from keras. And it is 60,000 for the training set while 10,000 for the test set.

Tried and tested the same code on Google Colab, Jupyter Notebook as well as Pycharm but the issue is still the same.
(Also, reshaped the input for the test and training set. And ran the code with old TensorFlow version 2.3.0 that was referenced in the book - Previously was using latest Tensor flow version 2.7.0)

[https://colab.research.google.com/drive/1hkcax1f-4_t5N7FI6sCTyD1zZaBfM92n?authuser=1#scrollTo=eIF4S7sdIP_K&line=2&uniqifier=1](url)
